### PR TITLE
Finish up roadmap to 1.7.0

### PR DESCRIPTION
**Pull request by deviousasti**
- Remove `AwaitThreshold`
- Target CK 1.7.0 and librdkafka 1.7.0

### CODE REVIEW

**Comment by deviousasti** [comment]
This closes out #48

File: src/FsKafka/FsKafka.fs [hunk 1]
```diff
@@ -254,10 +254,6 @@ module Core =
                 while Volatile.Read(&inFlightBytes) > minInFlightBytes && not ct.IsCancellationRequested do
                     showConsumerWeAreStillAlive ()
                 log.Information "Consumer resuming polling"
-        [<Obsolete "Please use overload with ?busyWork=None">]
-        // TODO remove ?busyWork=None in internal call when removing this overload
```
Comment (bartelink) [inline src/FsKafka/FsKafka.fs:258]: I think there is a change to be made for this - I believe the disambiguation now becomes redunant?

File: src/FsKafka/FsKafka.fsproj [hunk 2]
```diff
@@ -15,14 +15,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="librdkafka.redist" Version="[1.7.0]" />
```
Comment (bartelink) [inline src/FsKafka/FsKafka.fsproj:18]: Best to preserve the ordering in the file i.e. meta, F#, actual dependencies i.e.
```
Sourcelink
MinVer

FSharp.Core
````
then the remaining 4 after that

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,10 +14,12 @@ The `Unreleased` section name is replaced by the expected version of next releas
 
 ### Added
 ### Changed
+- Target `Confluent.Kafka [1.7.0]`, `librdkafka.redist [1.7.0]`
```
Comment (bartelink) [inline CHANGELOG.md:17]:
```suggestion
- Target `Confluent.Kafka [1.7.0]`, `librdkafka.redist [1.7.0]` [#89](https://github.com/jet/FsKafka/pull/89)
```

File: src/FsKafka/FsKafka.fs (hunk 1 above)
Comment (deviousasti) [reply src/FsKafka/FsKafka.fs:258]: There was no internal call referencing this overload.

File: src/FsKafka/FsKafka.fsproj (hunk 2 above)
Comment (deviousasti) [reply src/FsKafka/FsKafka.fsproj]: Changed back to original order.

**Pull request merged**